src/features/contentCreator/addPluginPage.ts

```diff
@@ -0,0 +1,397 @@
+                </div>
+
+                <div class="group-buttons">
+                  <vscode-button id="clear-button" form="init-form" appearance="secondary">
+                    <span class="codicon codicon-clear-all"></span>
+                    &nbsp; Clear All
+                  </vscode-button>
+                  <vscode-button id="create-button" form="init-form">
+                    <span class="codicon codicon-run-all"></span>
+                    &nbsp; Create
+                  </vscode-button>
+                </div>
+
+                <br>
+                <vscode-divider></vscode-divider>
+                <br>
+                <vscode-text-area id="log-text-area" cols="512" rows="10" placeholder="Output of the command execution"
+                  resize="vertical" readonly>Logs</vscode-text-area>
+
+                <div class="group-buttons">
+                  <vscode-button id="clear-logs-button" form="init-form" appearance="secondary">
+                    <span class="codicon codicon-clear-all"></span>
+                    &nbsp; Clear Logs
+                  </vscode-button>
+                  <vscode-button id="open-folder-button" form="init-form" disabled>
+                    <span class="codicon codicon-go-to-file"></span>
+                    &nbsp; Open Plugin
+                  </vscode-button>
+                </div>
+
+                <div id="required-fields" class="required-fields">
+                  <p>Fields marked with an asterisk (*) are required</p>
+                </div>
+
+              </section>
+            </form>
+
+          <!-- Component registration code -->
+          <script type="module" nonce="${nonce}" src="${webviewUri}"></script>
+        </body>
+      </html>
+    `;
+  }
+
+  private _setWebviewMessageListener(webview: vscode.Webview) {
+    webview.onDidReceiveMessage(
+      async (message: any) => {
+        const command = message.command;
+        let payload;
+
+        switch (command) {
+          case "open-explorer": {
+            payload = message.payload;
+            const selectedUri = await this.openExplorerDialog(
+              payload.selectOption,
+            );
+            webview.postMessage({
+              command: "file-uri",
+              arguments: { selectedUri: selectedUri },
+            } as PostMessageEvent);
+            return;
+          }
+          case "init-create":
+            payload = message.payload as PluginFormInterface;
+            await this.runAddCommand(payload, webview);
+            return;
+
+          case "init-open-scaffolded-folder":
+            payload = message.payload;
+            await this.openFolderInWorkspace(
+              payload.projectUrl,
+              payload.pluginName,
+              payload.pluginType,
+            );
+            return;
+        }
+      },
+      undefined,
+      this._disposables,
+    );
+  }
+
+  private async getCreatorVersion(): Promise<string> {
```

Comment:
This function can also be removed and imported from the utils file if needed anywhere.